Client/Frontend/Reader/ReaderModeStyleViewController.swift

```diff
@@ -165,71 +247,114 @@ class ReaderModeStyleViewController: UIViewController {
+        case .Serif:
+            setTitle("Serif", forState: UIControlState.Normal)
+            let f = ReaderModeStyleViewControllerUX.ThemeTitleFontSerif
+            titleLabel?.font = f
+        }
     }
 }
 
-/// Custom button that knows how to show the right image and selection style for a ReaderModeFontSize
-/// value. Very generic now, which will change when we have more final UX assets.
-class FontSizeButton: UIButton {
-    var fontSize: ReaderModeFontSize!
+// MARK: -
 
-    override init(frame: CGRect) {
-        super.init(frame: frame)
-    }
+enum FontSizeAction {
+    case Smaller
+    case Bigger
+}
 
-    required init(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
-    }
+class FontSizeButton: UIButton {
+    var fontSizeAction: FontSizeAction = .Bigger
 
-    convenience init(fontSize: ReaderModeFontSize) {
+    convenience init(fontSizeAction: FontSizeAction) {
         self.init(frame: CGRectZero)
-        self.fontSize = fontSize
-        setTitle("\(fontSize.rawValue)", forState: UIControlState.Normal)
-        setTitleColor(UIColor.blackColor(), forState: UIControlState.Selected)
-        setTitleColor(UIColor.lightGrayColor(), forState: UIControlState.Normal)
-        backgroundColor = UIColor.whiteColor()
+        self.fontSizeAction = fontSizeAction
+
+        setTitleColor(ReaderModeStyleViewControllerUX.FontSizeButtonTextColorEnabled, forState: UIControlState.Normal)
+        setTitleColor(ReaderModeStyleViewControllerUX.FontSizeButtonTextColorDisabled, forState: UIControlState.Disabled)
+
+        switch fontSizeAction {
+        case .Smaller:
+            setTitle("-", forState: .Normal)
+        case .Bigger:
+            setTitle("+", forState: .Normal)
+        }
+
+        // TODO Does this need to change with the selected font type? Not sure if makes sense for just +/-
+        titleLabel?.font = ReaderModeStyleViewControllerUX.FontSizeButtonTextFont
     }
 }
 
-/// Custom button that knows how to show the right image and selection style for a ReaderModeTheme
-/// value. Very generic now, which will change when we have more final UX assets.
-class ThemeButton: UIButton {
-    var theme: ReaderModeTheme!
+// MARK: -
 
-    override init(frame: CGRect) {
-        super.init(frame: frame)
+class FontSizeLabel: UILabel {
+    override convenience init() {
+        self.init(frame: CGRectZero)
+        text = "Aa"
     }
 
-    required init(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+    var fontType: ReaderModeFontType = .SansSerif {
+        didSet {
+            switch fontType {
+            case .SansSerif:
+                font = ReaderModeStyleViewControllerUX.FontSizeLabelFontSansSerif
+            case .Serif:
+                font = ReaderModeStyleViewControllerUX.FontSizeLabelFontSerif
+            }
+        }
     }
+}
+
+// MARK: -
+
+class ThemeButton: UIButton {
+    var theme: ReaderModeTheme!
```

Comment:
Argh. I hate these view initializers being screwed. Above you used a default value. Here you're using explicit unwrapping. Probably best to be consistent.